api/passthrough/getImageByAmiId.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var self = getImageByAmiId;
+module.exports = self;
+
+var async = require('async');
+var _ = require('underscore');
+
+var AWSAdapter = require('../../common/awsAdapter.js');
+
+function getImageByAmiId(req, res) {
+  var bag = {
+    reqQuery: req.query,
+    accessToken: req.query.accessToken,
```

Comment:
the tokens are already available in admiral.env file. why are they being passed in request query?